lib/database/adapters/LhcFillAdapter.js

```diff
@@ -49,9 +49,9 @@ class LhcFillAdapter {
 
         return {
             fillNumber,
-            stableBeamsStart: new Date(stableBeamsStart).getTime(),
-            stableBeamsEnd: new Date(stableBeamsEnd).getTime(),
-            stableBeamsDuration,
+            stableBeamsStart: stableBeamsStart ? new Date(stableBeamsStart).getTime() : stableBeamsStart,
+            stableBeamsEnd: stableBeamsEnd ? new Date(stableBeamsEnd).getTime() : stableBeamsEnd,
+            stableBeamsDuration: stableBeamsDuration ? new Date(stableBeamsDuration).getTime() : stableBeamsDuration,
```

Comment:
I think that the duration is not a Date

Reply:
you're right, it looks like copilot went along and changed that field as well. My mistake, I fixed it now. Sorry.